certified-connectors/Datablend/apiDefinition.swagger.json

```diff
@@ -0,0 +1,959 @@
+                "limit": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "limit",
+				  "default":1
+                },
+                "offset": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "offset",
+				  "default":0
+                },
+                "orders": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "path": {
+                        "type": "string",
+                        "description": "path",
+						"default":"created"
+                      },
+                      "direction": {
+                        "type": "string",
+                        "description": "direction",
+						"default":"Descending"
+                      }
+                    }
+                  },
+                  "description": "order by"
+                },
+                "predicate": {
+                  "type": "object",
+                  "properties": {
+                    "path": {
+                      "type": "string",
+                      "description": "path",
+					  "default":"parent.id"
+                    },
+                    "value": {
+                      "type": "string",
+                      "description": "id of the workflow previously executed"
+                    },
+                    "comparator": {
+                      "type": "string",
+                      "description": "comparator",
+					  "default":"EqualTo"
+                    }
+                  },
+                  "description": "predicate"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/workflow-executions": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+				"$ref": "#/definitions/WorkflowExecutions"
+			}
+          }
+        },
+        "summary": "Workflow Executions",
+        "description": "Workflow Executions",
+        "operationId": "WorkflowExecutions",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "parent": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "description": "ID of the workflow to execute"
+                    }
+                  },
+                  "description": "parent"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/workflows/search": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
```

Comment:
@bmp02050 Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger. **This applies to all the default responses in the swagger.**